packages/frontend/src/components/chart/configure/render/render.ts

```diff
@@ -64,6 +64,12 @@ export function render(
     )?.parentElement
     const toggle = !isTvl && !isDetailedTvl
 
+    elements.controls.chartType.forEach((input) => {
+      const isChecked =
+        input.id === `radio-chart-type-controls-${state.controls.view}`
+      input.checked = isChecked
+    })
+
```

Comment:
Nit pick: Let's move this above the `toggle` definition - a bit more logical code flow 
```suggestion

    elements.controls.chartType.forEach((input) => {
      const isChecked =
        input.id === `radio-chart-type-controls-${state.controls.view}`
      input.checked = isChecked
    })

    const toggle = !isTvl && !isDetailedTvl
```